src/plugins/template/backend/int_executable.cpp

```diff
@@ -159,8 +145,13 @@ bool runtime::interpreter::INTExecutable::call(const vector<shared_ptr<runtime::
             }
         }
 
+        const auto tensor_inputs = ov::util::wrap_tensors(op_inputs);
+        auto tensor_outputs = ov::util::wrap_tensors(op_outputs);
+
         // Call evaluate for cloned_node with static shapes
-        if (!cloned_node->evaluate(op_outputs, op_inputs, eval_context)) {
+        if (cloned_node->evaluate(tensor_outputs, tensor_inputs, eval_context)) {
+            ov::util::update_output_host_tensors(op_outputs, tensor_outputs);
+        } else {
             evaluate_node(cloned_node, op_outputs, op_inputs);
```

Comment:
what is `eval_context`? Would it affect [evaluate_map](https://github.com/openvinotoolkit/openvino/blob/91df0a8aa92dbcf67c85676e9a786cc360486b17/src/plugins/template/backend/evaluates_map.cpp#L4) to get reference results?

Reply:
It looks like it is used by operators ReadValue, Assign. Current changes don't change the way how it is used.

@ilyachur 
Can you explain purpose of this `context`?

Reply:
Evaluation context is a runtime information which is needed for evaluate some operations. It is used in some memory ops like read value.